### DISCUSSION TITLE
Auto-Syncing New Folders

### DISCUSSION THREAD

**Discussion by iamjddavis** [samschott/maestral#601] [Q&A]
I'm having an issue where Maestral automatically syncs new folders created on the dropbox website. I have a limited number of folders syncing via the Selective Sync folder, but new folders created outside of those folders (on the Dropbox website) will sometimes still sync local via Maestral.

Is there any way to turn this off or make sure only folders chosen in Selective Sync are downloaded?

**Comment by samschott** [comment]
Unfortunately not, at least at the moment, but I'll take it as a feature request. Do you imagine this working for top-level folders only or at every level?

**Reply by iamjddavis** [reply]
I guess at every level. I only first noticed it on top-level folders a few weeks ago and in my situation those aren't created very often so I thought I could work around it, but it seems like this week some sub-folders were syncing (which is what caught my attention). I may be an edge case with my selective sync choices, but if you're able to add a way to disable auto-syncing in the future I'd appreciate it. Thanks for your work on everything.

**Reply by jiehanzheng** [reply]
I would love this feature (make Selective Sync inclusion-based, not exclusion based).  Top level is fine for me -- better than nothing! :)

**Comment by idachev-shipcars** [comment]
Hello everyone!

I had the same issue that my Dropbox span 1Tb and I could not fit in one of my disks.

So I start digging for a solution :) and found this wonderful tool: mergerfs
https://github.com/trapexit/mergerfs

Which on Linux can make a FUSE mount that merges multiple folders span on different partitions.

The official Dropbox client refused to work with FUSE so I end up with maestral and so far it is the best tool I integrated into my work envs for years!
